### PR TITLE
{bio}[intel/2017b] ClustalW2 v2.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/91ba7731cad9537e40532fdfba1d9955 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2151.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/58cfac5bc1fd74c0013bb0c5108b91ce for a full test report.

File: easybuild/easyconfigs/c/ClustalW2/ClustalW2-2.1-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name = 'ClustalW2'
+version = '2.1'
+
+homepage = 'http://www.ebi.ac.uk/Tools/msa/clustalw2/'
+description = """ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['ftp://ftp.ebi.ac.uk/pub/software/%s/%s' % (name.lower(), version)]
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/ClustalW2/ClustalW2-2.1-intel-2017b.eb]: any reason not to use `%(namelower)s` and friends?

Comment (boegel) [reply easybuild/easyconfigs/c/ClustalW2/ClustalW2-2.1-intel-2017b.eb]: Will fix, thanks.

**Pull request merged**